vars/common-vars.yml

```diff
@@ -1,10 +1,21 @@
 ---
-upstream_repos_clone_folder: /opt
-developer_mode: "{{ (lookup('env','OSH_DEVELOPER_MODE') | default('False', true) ) | bool }}"
-socok8s_workspace: "{{ lookup('env','ANSIBLE_RUNNER_DIR') | default('~/suse-osh-deploy', true) }}"
-socok8s_extravars: "{{ socok8s_workspace }}/env/extravars"
-socok8s_ses_pools_details: "{{ socok8s_workspace }}/ses_config.yml"
-socok8s_registry_cert: "{{ socok8s_workspace }}/certs/domain.crt"
-socok8s_registry_certkey: "{{ socok8s_workspace }}/certs/domain.key"
-socok8s_caasp_environment_details: "{{ socok8s_workspace }}/environment.json"
-socok8s_libvirtuuid: "{{ socok8s_workspace }}/libvirt.uuid"
+# Proposing to phase out common_vars in favor of group_vars to define usage of global
+# variables and auto loading/availability of variables in roles
```

Comment:
Please do this in another commit.

As said above, I don't agree with that in the same commit, and on top of it, I think it's a very bad idea.

Reply:
Updated file and should not impact osh-deployment.